test/config/jest-playwright.js

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-const { isRunningOnCISlowOS, isRunningOnCi } = require('../helpers/environment-utils');
+const { isRunningOnCISlowOS, isRunningOnCi } = require('../shared/environment-utils');
```

Comment:
**suggestion:** 
```suggestion
const { isRunningOnCISlowOS, isRunningOnCi } = require('@test/shared/environment-utils');
```

Reply:
❌ This won't work here. The "path" is only for TypeScript code.
I tested it locally and got this error `Error: Jest: Got error running globalSetup - <redacted>/bpmn-visualization-js/node_modules/jest-playwright-preset/setup.js, reason: Cannot find module '@test/shared/environment-utils'`